jest.preset.js

```diff
@@ -4,10 +4,16 @@ const path = require('path');
 
 module.exports = {
   ...nxPreset,
-  testEnvironment: path.join(
-    __dirname,
-    'scripts/gf/test-util',
-    'jsdom-lax-ssl-environment.ts'
-  ),
+  testEnvironment: path.join(__dirname, 'scripts/gf/test-util', 'jsdom-lax-ssl-environment.ts'),
   testURL: 'https://localhost:5001',
```

Comment:
If we change this, it will properly cause the need to manually migrate in later versions since it's not NX standard, so I wouldn't recommend changing this.